### PR TITLE
Bump build number

### PR DESCRIPTION
**Pull request by jschueller**
@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge/core, yes rerendering wipes the appveyor config and disables the windows build

**Comment by djsutherland** [comment]
This is because of changes related to how VC versions work. The new system to use is outlined at https://conda-forge.org/docs/buildwin.html#building-for-different-vc-versions; I'll send a PR right now updating it to the current system.

Yes, we know this should have been communicated better. :) If you have suggestions on how to do that feel free to weigh in: https://github.com/conda-forge/conda-forge.github.io/issues/578.

**Comment by jschueller** [comment]
Ok, thanks, I'll try to update.

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,14 +12,11 @@ source:
...
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and py>=35]
-  skip: true  # [win and py36]
+  number: 4
 
 requirements:
   build:
+    - {{ compiler('cxx') }}
     - toolchain
```
Comment (djsutherland) [inline recipe/meta.yaml:20]: Don't need this explicitly anymore if you're using `{{ compiler() }}`; it's pulled in automatically when appropriate and not used when not needed.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,14 +12,11 @@ source:
     - no-console.patch
 
 build:
-  number: 2
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and py>=35]
-  skip: true  # [win and py36]
+  number: 4
```
Comment (djsutherland) [inline recipe/meta.yaml:15]: No need to bump the number for PRs that never merged; 3 is fine. :)

**Comment by djsutherland** [comment]
Oh, sorry, wasn't clear enough that I was also doing it.

We could use your version instead, doesn't matter....

File: recipe/meta.yaml (hunk 2 above)
Comment (jschueller) [reply recipe/meta.yaml:15]: Revision 3 was published as I merged accidentally :(

**Pull request closed without merging**